modules/gitops-cluster-scoped-instance.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * multitenancy/multitenancy-support-in-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-cluster-scoped-instance_{context}"]
+= Cluster-scoped instance 
+
+A cluster-scoped instance is intended to deploy and manage resources across a cluster. Cluster-scoped instances have access to cluster-level resources and thus are typically, but not always, used for cluster configuration. You can choose to elevate certain namespace-scoped Argo CD instances to become cluster-scoped. To elevate the instances, you must modify the `Subscription` resource of the {gitops-shortname} Operator.
```

Comment:
Do we have documentation on how this elevation can be achieved with a Subscription ?

Reply:
Yes, have added in the Additional resources section.